core/templates/services/contextual/url.service.ts

```diff
@@ -22,6 +22,10 @@ import { Injectable } from '@angular/core';
 
 import { WindowRef } from 'services/contextual/window-ref.service';
 
+interface UrlParams {
+  [param: string]: string
```

Comment:
Does this signify that UrlParams can have any number of params all with type string?

Reply:
@bansalnitish Yup, it acts like a dict whose keys and values are both dict.

Reply:
Add a comment to explain. In general - if you think a reviewer might question you on something, it is always a good idea to add a comment beforehand.

Reply:
Ditto.

Reply:
Done